lte/cloud/go/services/subscriberdb/servicers/southbound/subscriberdb_servicer.go

```diff
@@ -271,7 +300,15 @@ func (s *subscriberdbServicer) loadSubscribersPageFromCache(ctx context.Context,
 		return nil, "", err
 	}
 
-	return subProtos, nextToken, nil
+	cloudflag := GetCloudSubscriberDbEnabled(ctx)
+	if cloudflag {
+		glog.V(2).Infof("Cloud Authentication enabled, not streaming subscriber data")
+		return nil, "", nil
```

Comment:
same, move it up to the beginning, no reason to compile the page if it's not needed